controllers/handler/deployment.go

```diff
@@ -356,8 +356,11 @@ func (ingressRequest *IngressRequest) CreateOrUpdateDeployment(clusterType strin
 	image := os.Getenv("ICP_MANAGEMENT_INGRESS_IMAGE")
 	var hostHeader string
 	if clusterType == CNCF {
+		dn := ingressRequest.managementIngress.Status.Host
 		pos := strings.LastIndex(ingressRequest.managementIngress.Status.Host, ":")
-		dn := ingressRequest.managementIngress.Status.Host[0:pos]
+		if pos >= 0 {
```

Comment:
It's when no port provided?

Reply:
Correct, this covers the condition when no port is specified.  We have another PR that always adds the port for CNCF, so in theory it should never happen, but I didn't want to leave the code in this condition ... cause, you never know :)